### PR TITLE
20MHz JTAG VCU118

### PR DESCRIPTION
**Pull request by erikdanie**
Now that JTAG is moving to be less speed restricted, up the VCU118 default JTAG speed

### CODE REVIEW

**Review by ernie-sifive — APPROVED** [review]

File: generate_openocdcfg.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def main(argv):
         parsed_args.dts, followIncludes=True)
 
     if "vcu118" in parsed_args.board:
-        adapter_khz = 1800
+        adapter_khz = 20000
     else:
         adapter_khz = 10000
```
Comment (sifivekevin) [inline generate_openocdcfg.py:128]: Any reason we should just use 20 across the board?

**Review by sifivekevin — APPROVED** [review]
LGTM

File: generate_openocdcfg.py (hunk 1 above)
Comment (erikdanie) [reply generate_openocdcfg.py:128]: (assuming you mean shouldn't)
I'm not sure that I've ever tested faster than 10 on arty, but we could if it's known to work

Comment (sifivekevin) [reply generate_openocdcfg.py:128]: Let it lie, especially since Arty is being "deprecated".

**Review by bsousi5 — APPROVED** [review]
LGTM

**Pull request merged**